cmd/falco/runner_test.go

```diff
@@ -88,6 +88,25 @@ func TestResolveWithDuplicateDeclarations(t *testing.T) {
 	}
 }
 
+func TestResolveWithMultipleModules(t *testing.T) {
+	fileName := "../../terraform/data/terraform-modules.json"
+	rslv, f := loadFromTfJson(fileName, t)
+
+	r, err := NewRunner(&Config{V: true}, f)
+	if err != nil {
+		t.Fatalf("Unexpected runner creation error: %s", err)
+	}
+
+	ret, err := r.Run(rslv[0])
+	if err != nil {
+		t.Fatalf("Unexpected Run() error: %s", err)
+	}
+
+	if ret.Errors > 0 {
+		t.Errorf("Errors expects 0, got %d", ret.Errors)
+	}
+}
+
```

Comment:
Could you add more test cases to include module without `.vcl` extension please?